### ISSUE TITLE
Min and Max as config?

### ISSUE THREAD

**Issue by brownad** [luc-ass/homebridge-evohome#37]
Would it be possible to move the min and max setting for thermostats to config instead?  I already set mine to a min of 10c on Evohome.

**Comment by fredericvl** [comment]
Maybe we should go for a better solution and get the min & max values from honeywell?
I see it's possible, each zone has something like this:

```
"setpointCapabilities": {
    "maxHeatSetpoint": 35,
    "minHeatSetpoint": 5,
    "valueResolution": 0.5,
    "canControlHeat": true,
    "canControlCool": false,
    "allowedSetpointModes": [
        "PermanentOverride",
        "FollowSchedule",
        "TemporaryOverride"
    ],
    "maxDuration": "1.00:00:00",
    "timingResolution": "00:10:00"
}
```

I'll have a look at it in the coming days.

**Comment by brownad** [comment]
Now that would be even cooler 😎

**Comment by luc-ass** [comment]
The issue has been resolved by @fredericvl in commit #38 

Merged.
Bumped version to v0.5.0.
Published to NPM.
